pkg/env/dev.go

```diff
@@ -77,6 +78,15 @@ func newDev(ctx context.Context, log *logrus.Entry, instancemetadata instancemet
 		}
 	}
 
+	var err error
+	rpHome := os.Getenv("RP_HOME")
```

Comment:
https://stackoverflow.com/questions/31873396/is-it-possible-to-get-the-current-root-of-package-structure-as-a-string-in-golan

Reply:
@asalkeld how's this look?